src/solana/SolanaEngine.ts

```diff
@@ -252,11 +255,12 @@ export class SolanaEngine extends CurrencyEngine<
 
   async queryFee(): Promise<void> {
     try {
-      const response = await this.fetchRpc('getRecentBlockhash')
-      const {
-        feeCalculator: { lamportsPerSignature }
-      } = asRecentBlockHash(response).value
-      this.feePerSignature = lamportsPerSignature.toString()
+      const response = await this.fetchRpc('getRecentPrioritizationFees')
+      const recentPriorityFees = asGetRecentPrioritizationFees(response)
+      const latestPriorityFee = recentPriorityFees.sort(
+        (a, b) => a.slot - b.slot
```

Comment:
This could crash if the array is empty, but since we are in a try block anyhow, I guess it doesn’t matter. Might warrant a comment, though.